modules/sugar/src/test/ts/browser/ElementInstancesTest.ts

```diff
@@ -0,0 +1,25 @@
+import { UnitTest, Assert } from '@ephox/bedrock-client';
+import Element from 'ephox/sugar/api/node/Element';
+import { tElement, eqElement } from 'ephox/sugar/test/ElementInstances';
+import { Option, OptionInstances } from '@ephox/katamari';
+import tOption = OptionInstances.tOption;
```

Comment:
Another place where you've used that `import` syntax instead of using a `const`?

Reply:
Thanks. Fixed.